jormungandr/src/explorer/indexing.rs

```diff
@@ -89,6 +94,33 @@ pub enum ExplorerAddress {
     Old(OldAddress),
 }
 
+#[derive(Clone)]
+pub struct ExplorerVotePlan {
+    pub id: VotePlanId,
+    pub vote_start: BlockDate,
+    pub vote_end: BlockDate,
+    pub committee_end: BlockDate,
+    pub payload_type: PayloadType,
+    pub proposals: Vec<ExplorerVoteProposal>,
+}
+
+#[derive(Clone)]
+pub struct ExplorerVoteProposal {
+    pub proposal_id: ExternalProposalId,
+    pub options: Options,
+    pub tally: Option<ExplorerVoteTally>,
+    pub votes: std::collections::HashMap<ExplorerAddress, Choice>,
```

Comment:
I'm curious about this `HashMap` . Is it not another `Hamt` for a reason? Is it fine to clone it on every vote?